PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-resource-connector-lib/src/main/java/com/microsoft/azure/toolkit/intellij/connector/dotazure/AzureModule.java

```diff
@@ -78,7 +78,7 @@ public AzureModule(@Nonnull final Module module, @Nullable VirtualFile dotAzure)
 
     @SneakyThrows(value = {IOException.class, JDOMException.class})
     private void loadProfiles(@Nonnull final VirtualFile profilesXmlFile) {
-        if (profilesXmlFile.contentsToByteArray().length < 1) {
+        if (profilesXmlFile.contentsToByteArray().length < 1 && !profilesXmlFile.exists()) {
```

Comment:
should be `!profilesXmlFile.exists() || profilesXmlFile.contentsToByteArray().length < 1`?